### ISSUE TITLE
Rule proposal: Forbid the use `.always` when not used with `after`/`afterEach`

### ISSUE THREAD

**Issue by jfmengels** [avajs/eslint-plugin-ava#139]
Rule proposal: Forbid the use `.always` when not used with `after`/`afterEach`

I'm not sure what it does currently when you use `always` without the `after` modifier, but it is at least useless and confusing.
### Fail

``` js
test.always(fn);
test.cb.always(fn);
test.always.cb(fn);
```
### Valid

``` js
test.cb(fn);
test.after.always(fn);
test.always.after(fn);
test.afterEach.always(fn);
test.always.afterEach(fn);
```

**Comment by sindresorhus** [comment]
👍
